x-pack/test/security_solution_endpoint/services/endpoint.ts

```diff
@@ -77,6 +78,27 @@ export class EndpointTestResources extends FtrService {
     await this.transform.api.updateTransform(transform.id, { frequency }).catch(catchAndWrapError);
   }
 
+  private async stopTransform(transformId: string) {
+    const stopRequest = {
+      transform_id: `${transformId}*`,
+      force: true,
+      wait_for_completion: true,
+      allow_no_match: true,
+    };
+    return this.esClient.transform.stopTransform(stopRequest);
+  }
+
+  private async startTransform(transformId: string) {
+    const transformsResponse = await this.esClient.transform.getTransform({
+      transform_id: `${transformId}*`,
+    });
+    return Promise.all(
+      transformsResponse.transforms.map((transform) => {
```

Comment:
I assume the number of transforms here is small and thus we don't need to throttle the calls to `*.startTranform()` - right?

Reply:
Yep. We control these transforms and it's only 2 so no worries there.